tvalmetrics/rag_scores_calculator.py

```diff
@@ -459,10 +494,11 @@ def get_n_samples(
             raise ValueError(error_message)
         return n_samples
 
-    def score_batch(
+    def __score_batch(
```

Comment:
Why did you make this a private method? It's meant to be exposed to the user and is never used anywhere in this class. I'm going to change it back to non private method.